src/main/java/frc/team2412/robot/subsystems/DrivebaseSubsystem.java

```diff
@@ -20,21 +20,22 @@
 import swervelib.SwerveDrive;
 import swervelib.math.SwerveMath;
 import swervelib.parser.SwerveParser;
-import swervelib.telemetry.SwerveDriveTelemetry;
 
 public class DrivebaseSubsystem extends SubsystemBase {
 
 	// SWERVE CONSTANTS (that aren't in deploy dir)
 
-	private static final double MAX_SPEED = 0.1;
+	private static final double MAX_SPEED = 2.0;
 	private static final double JOYSTICK_DEADBAND = 0.05;
-	private static final double DRIVEBASE_RADIUS = 0;
+	private static final double DRIVEBASE_RADIUS =
+			Math.hypot(8.5, 8.5); // our wheels are 8.5 inches by 8.5 inches from the center of the bot;
```

Comment:
Should this be calculated from `SwerveDrive.swerveDriveConfiguration.moduleLocationsMeters` (because we have different robots)?